### ISSUE TITLE
[Docs] Meshery Development Incompatible

### ISSUE THREAD

**Issue by samyakshah3008** [meshery/meshery#7885]
#### Current State

After following all the steps mentioned to run the server, many are still facing below error: 

![image](https://github.com/meshery/meshery/assets/86982322/22177454-4b04-4c01-aab6-546ff9d4a318)

#### Desired State

As resolved by @Abhishek-kumar09  Solution to this particular issue is by: 

### [solution in discuss.layer5.io](https://discuss.layer5.io/t/meshery-ui-dev-server-throwing-error/2873/5?u=abhi)

---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

**Comment by samyakshah3008** [comment]
I would love to go ahead and fix it.

**Comment by leecalcote** [comment]
Related to https://github.com/meshery/meshery/issues/7850

**Comment by samyakshah3008** [comment]
Hey @Abhishek-kumar09 I was facing one blocker:

I know how I am going to write the problem statement with answer, but unsure about the "exact" location where I can add this error with it's solution, can you guide me here please?

By exact location I mean, as we have whole meshery docs, I faced this issue while starting the make server so shall I add this point in servers page? Also shall we add the same in the contributing guide dot md file as well? 

![Screenshot_2023_0621_103447.jpg](https://github.com/meshery/meshery/assets/86982322/a5b4f591-342b-4156-abd5-902fa2854531)

**Comment by vaibhav-xt** [comment]
@samyakshah3008 
Adding an issue to the Meshery server page is a good idea 💡. Every person, regardless of whether they use Windows, Linux, or macOS, will refer to this page after setting up their environment to learn how to run the server. Whether they are a Windows, macOS, or Linux user, if they encounter this issue, they can find the solution instantly.

**Comment by samyakshah3008** [comment]
Update: 
Will raise a PR tomorrow to fix this issue 

Blockers: 
University exams

// @Abhishek-kumar09

**Comment by samyakshah3008** [comment]
Hi @Abhishek-kumar09 I have raised a PR in which I have updated the contributing dot md file. 
However, we are still left to update the same content on Meshery docs. 

Blockers: 
- The reason I have not updated the content on Meshery docs was as ruby version recently got updated, I am facing setup issues while started the Meshery docs. 
- As this task is already delayed due to my University exams came in the middle, I feel we can assign the remaining portion to someone else so this issue ticket can be fixed to the soonest. 
- Meanwhile I will figure out to fix setup issues with Meshery docs on my machine.
